src/Ryujinx.Horizon/Sdk/Friends/Detail/Ipc/FriendsServicePermissionLevel.cs

```diff
@@ -1,16 +1,13 @@
-using System;
-
-namespace Ryujinx.HLE.HOS.Services.Friend.ServiceCreator
+namespace Ryujinx.Horizon.Sdk.Friends.Detail.Ipc
 {
-    [Flags]
-    enum FriendServicePermissionLevel
+    enum FriendsServicePermissionLevel
     {
         UserMask = 1,
         ViewerMask = 2,
         ManagerMask = 4,
         SystemMask = 8,
```

Comment:
```suggestion
        ViewerMask = 3,
        ManagerMask = 7,
        SystemMask = 9,
```

![image](https://github.com/Ryujinx/Ryujinx/assets/4905390/5cde6697-cb92-4188-b886-7145bba5d5fd)

Reply:
The actual values are below:
```cs
        Admin = -1,
        User = UserMask,
        Viewer = UserMask | ViewerMask,
        Manager = UserMask | ViewerMask | ManagerMask,
        System = UserMask | SystemMask,
```
so User = 1, Viewer = 1 | 2 = 3, Manager = 1 | 2 | 4 = 7 and System is 1 | 8 = 9. So it should be already correct.

Reply:
Oh yeah my bad I didn't see them sorry ^^'